tools/coreml/test/test_mxnet_converter.py

```diff
@@ -444,17 +473,39 @@ def test_tiny_conv_random_input_multi_filter(self):
         )
         self._test_mxnet_model(net, input_shape=input_shape, mode='random')
 
+    def test_tiny_conv_random_input_multi_group(self):
+        np.random.seed(1988)
+        input_shape = (1, 16, 10, 10)
+        num_filter = 16
+        num_group = 4
+        kernel = (5, 5)
+        stride = (1, 1)
+        pad = (0, 0)
+        net = mx.sym.Variable('data')
+        net = mx.symbol.Convolution(
+            data=net,
+            num_filter=num_filter,
+            num_group=num_group,
+            kernel=kernel,
+            stride=stride,
+            pad=pad,
+            name='conv_1'
+        )
+        self._test_mxnet_model(net, input_shape=input_shape, mode='random')
+
     def test_conv_random(self):
         np.random.seed(1988)
         input_shape = (1, 3, 10, 10)
         num_filter = 64
+        num_group = 1
```

Comment:
Does this test case fail if num_group is not provided? 
If it doesn't fail then it might be better to not explicitly add num_group.

Reply:
It doesn't fail.
I will rollback this change.

Reply:
Done in 0189933